chef/cookbooks/glance/recipes/ceph.rb

```diff
@@ -24,20 +27,22 @@
 else
   # If external Ceph cluster will be used,
   # we need install ceph client packages
-  if node[:platform] == "suse" && File.exists?("/etc/ceph/ceph.client.admin.keyring")
-    package "ceph-common"
-    package "python-ceph"
-  else 
+  if File.exists?(ceph_conf) && File.exists?(admin_keyring)
+    if node[:platform] == "suse"
+      package "ceph-common"
+      package "python-ceph"
+    end
+  else
     return
```

Comment:
I submitted https://github.com/mjura/barclamp-glance/pull/2/files?w=1 to your branch as an apology :-)